finagle/h2/src/main/scala/com/twitter/finagle/buoyant/h2/Message.scala

```diff
@@ -90,6 +90,12 @@ trait Request extends Message {
   def authority: String
   def path: String
   override def dup(): Request
+  def transform(
```

Comment:
that looks wrong on so many levels... but if you insist I could do that

a mutable field of an immutable object is asking for trouble if you ask me